src/Microsoft.Health.Fhir.Core/Features/Conformance/SystemConformanceProvider.cs

```diff
@@ -98,6 +104,38 @@ public override async Task<ResourceElement> GetCapabilityStatementOnStartup(Canc
             return _listedCapabilityStatement;
         }
 
+        public async void RebuildCapabilityStatement()
+        {
+            while (_runRebuilder)
+            {
+                Thread.Sleep(_rebuildDelay);
```

Comment:
Should we use task delay instead here?
await Task.Delay(_rebuildDelay).ConfigureAwait(false);

Reply:
Replaced